client/components/Events/EventScheduleSummary/index.tsx

```diff
@@ -6,17 +6,30 @@ import {RepeatEventSummary} from '../RepeatEventSummary';
 import {Row} from '../../UI/Preview';
 import {gettext, eventUtils, timeUtils} from '../../../utils';
 import './style.scss';
+import {IEventItem} from 'interfaces';
 
 
-export const EventScheduleSummary = ({schedule, noPadding, forUpdating, useEventTimezone}) => {
-    if (!schedule)
+interface IProps {
+    schedule: Partial<IEventItem>,
+    noPadding?: boolean,
+    forUpdating?: boolean,
+    useEventTimezone?: boolean
+}
+
+export const EventScheduleSummary = ({
+    schedule: event,
```

Comment:
If we're now passing in the entire Event, it would make more sense to rename this prop to `event` or `item` instead (as passing the entire Event into a prop named `schedule` can be misleading)